### ISSUE TITLE
Domoticz show loading

### ISSUE THREAD

**Issue by bricololo44** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#342]
**Issue description**
I have 2 domoticz in the same version V2020.2
However I notice a difference between the 2 versions.
For one the version is written V2020.2 in large characters and the other 2020.2 (without the V) and in very small characters.
My problem is that for version V2020.2, the plugin does not work with Chrome (only firefox).
The version 2020.2 works well with Chrome.

Here are the differences :
 -Domoticz1 version 2020.2    Rpi2    zigbee2mqtt@1.9.0       Works well
 -Domoticz2 version V2020.2  Rpi3    zigbee2mqtt@1.14.0     Works only with firefox (Other show Loading)

**Additional information**

Zigbee2MQTT version:
zigbee2mqtt@1.9.0   and   zigbee2mqtt@1.14.0 

Python version:  Python 3.7.3 for the both

Domoticz version:  2020.2  and V2020.2

Plugin version: ???

**Comment by bricololo44** [comment]
Rectification,
my domoticz2 appears with the same size of character with firefox.
Here are the differences :
-Domoticz1 version 2020.2 Rpi2 zigbee2mqtt@1.9.0       Works well
-Domoticz2 version 2020.2 Rpi3 zigbee2mqtt@1.14.0 👍 
      - With Firefox works well (same presentation as domoticz1)
      - With Chrome plugin does not work well (the version is written V2020.2 in large characters)

**Comment by stas-demydiuk** [comment]
Do you see any errors in browser console?

**Comment by bricololo44** [comment]
There is no error in browser, however the plugin interface does not work. (Just Loading...)
I just noticed that the problem is related to one of my Rpi3 b +.
When I move the SD card from one Rpi that works on the 2nd, then it's not works.
I don't understand why one Rpi works and the other doesn't.
And why does it work with Firefox and not chrome ?
And it's the same SD card (with Domoticz and Zigbee plugin).
Maybe it is because I did the complete installation from the first (however the device works on the 2nd).
